index.html

```diff
@@ -78,40 +78,45 @@
 		<th width=33%> Developer Resources </th>
 	</tr>
         <tr align="center">
-		<td> <a href="https://wiki.archlinux.org/index.php/ZFS">Arch</a> </td>
+		<td> <a href="https://wiki.alpinelinux.org/w/index.php?search=zfs">Alpine</a> </td>
```

Comment:
Are you intending to create a page here, this link seems to only point to search results.